sites/fast-creator/app/index.tsx

```diff
@@ -12,7 +12,7 @@ root.setAttribute("id", "root");
 document.body.appendChild(root);
 const style: HTMLStyleElement = document.createElement("style");
 style.innerText =
-    "@font-face { font-family: SegoeUIVF; src: url(https://res.cloudinary.com/fast-dna/raw/upload/v1558051831/SegoeUI-Roman-VF_web.ttf) format(\"truetype\"); font-weight: \"1 1000\"; } @font-face { font-family: Segoe UI; src: url('//c.s-microsoft.com/static/fonts/segoe-ui/west-european/Normal/latest.woff2') format('woff2'); } body, html { font-family: SegoeUIVF, Segoe UI, SegoeUI, Helvetica Neue, Helvetica, Arial, sans-serif; font-size: 12px; padding: 0; margin: 0; }";
```

Comment:
presumably someone needs to update the files pointing to cloudinary

Reply:
If this was the old CDN we used under Jason's account, then yes, this should be changed to use our own CDN, and place these files in source code `/site-utilities/statics/assets/fonts/*`